pkgs/development/libraries/libjpeg/default.nix

```diff
@@ -1,23 +1,34 @@
 { lib, stdenv, fetchurl, static ? false }:
 
-with lib;
-
-stdenv.mkDerivation {
-  name = "libjpeg-9d";
+stdenv.mkDerivation rec {
+  pname = "libjpeg";
+  version = "9d";
 
   src = fetchurl {
-    url = "http://www.ijg.org/files/jpegsrc.v9d.tar.gz";
+    url = "http://www.ijg.org/files/jpegsrc.v${version}.tar.gz";
     sha256 = "1vkip9rz4hz8f31a2kl7wl7f772wg1z0fg1fbd1653wzwlxllhvc";
   };
 
-  configureFlags = optional static "--enable-static --disable-shared";
+  configureFlags = lib.optional static "--enable-static --disable-shared";
 
   outputs = [ "bin" "dev" "out" "man" ];
 
-  meta = {
-    homepage = "http://www.ijg.org/";
+  preConfigure = ''
+    # Workarounds for the ancient libtool shipped by libjpeg.
+    ln -s $libtool/bin/libtool .
+    cp $libtool/share/libtool/config.guess .
+    cp $libtool/share/libtool/config.sub .
+  '';
+
+  preInstall = ''
+    mkdir -p $out/bin $out/lib $out/include $out/man/man1
+  '';
```

Comment:
builder.sh was never referenced by default.nix (https://github.com/NixOS/nixpkgs/blob/f1c8cbed524b756abd2cca366dbd06d9e224be77/pkgs/development/libraries/libjpeg/62.nix), why do you add this, it can't possibly work